src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -767,3 +797,16 @@ export const focusSearchListCommand: ICommandHandler = accessor => {
 		}
 	});
 };
+
+function getElementsToOperateOn(viewer: WorkbenchObjectTree<RenderableMatch, void>, currElement: RenderableMatch): { result: RenderableMatch[]; usingMultiselect: boolean } {
+	let elementsToRemove: RenderableMatch[] = viewer.getSelection().filter((x): x is RenderableMatch => x !== null);
```

Comment:
Do you know what nulls mean here?

Reply:
Not just "to remove"

Reply:
@joaomoreno Do you know why ObjectTree says that its AbstractTree super class has to have nullable elements? https://github.com/microsoft/vscode/blob/d5d5cbaa11aecbfbc22ba7f431a20fdcdfca5ef6/src/vs/base/browser/ui/tree/objectTree.ts#L46

Reply:
Basically the `null` element is the root. It's the reason why `setChildren(null, [...])` replaces the root's elements.

Reply:
Ah, that's right. Thanks.

@andreamah the code you have here is good, even though getSelection will probably never give us an array with a null.